common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -188,3 +188,9 @@ def definition_to_xml(self, resource_fs):
         for child in self.get_children():
             self.runtime.add_block_as_child_node(child, xml_object)
         return xml_object
+
+    @property
+    def non_editable_metadata_fields(self):
+        non_editable_fields = super(SequenceDescriptor, self).non_editable_metadata_fields
+        non_editable_fields.extend([SequenceFields.is_entrance_exam, ])
```

Comment:
Okay, then I rescind my previous position and join the majority in the decision to move this field to Scope.content :)  Will make the necessary change(s) now.